tests/ha/hawk_gui.pm

```diff
@@ -49,7 +49,8 @@ sub run {
     my $docker_image = "registry.opensuse.org/home/rbranco/branches/opensuse/templates/images/tumbleweed/containers/hawk_test";
     assert_script_run("docker pull $docker_image", 240);
 
-    select_console 'x11';
+    select_console('displaymanager');
+    $self->handle_displaymanager_login();
```

Comment:
Would this problem really appear only in "ha/hawk_gui"? Maybe we need to handle the "login" in activate_console instead?

Reply:
True, that will appear in any test that performs tasks on a text based console and then tries to select a console.

But, not sure about implementing it in activate_console...
:thinking: 

Maybe there is a case where you want to select a console but not login. Like checking number of users (multi_users_dm), like checking menu on GDM, like changing desktop (GNOME, KDE) on GDM.
Then it is resposibility of the test to perform a login or not.

WDYT?

Reply:
Hm, don't we already distinguish between the consoles "x11" and "displaymanager" for this? like "x11" when you want (or prefer?) a logged in console and "displaymanager" for not logged in. I guess by default `select_console` should always give you a logged in console. We can either have a specific, not-logged in console or we have a flag to pass `no_login` (maybe we already have that somewhere?) or we use a specific `select_not_logged_in_displaymanager` method that does this.

Reply:
> Would this problem really appear only in "ha/hawk_gui"? Maybe we need to handle the "login" in activate_console instead?

Agree. Judging by the comment on the bug, this would happen in any test that skips the `x11` login after boot, does some stuff in a term, and then gets back to `x11`.